skimage/registration/tests/test_phase_cross_correlation.py

```diff
@@ -263,6 +263,6 @@ def test_disambiguate_zero_shift():
     """
     image = camera()
     computed_shift, _, _ = phase_cross_correlation(
-            image, image, disambiguate=True, return_error='always'
+            image, image, disambiguate=disambiguate, return_error='always'
             )
-    assert computed_shift == (0, 0)
+    np.testing.assert_array_equal(computed_shift, np.array((0., 0.)))
```

Comment:
This test is not actually testing for the fix! `assert_array_equal` is not testing the type of the array as it calls `asanyarray` internally before checking anything! So you need to add an additional line to check if the type is correct.